api/app/controllers/spree/api/base_controller.rb

```diff
@@ -3,9 +3,6 @@
 module Spree
   module Api
     class BaseController < ActionController::Base
-      prepend_view_path Rails.root + "app/views"
-      append_view_path File.expand_path("../../../app/views", File.dirname(__FILE__))
```

Comment:
Just a relic from the era that BaseController inherited from Metal instead of Base